etl/steps/data/garden/oecd/2025-02-25/social_expenditure.py

```diff
@@ -0,0 +1,58 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+# Define indicator columns and their new names.
+INDICATOR_COLUMNS = {
+    "Percentage of GDP": "share_of_gdp",
+    "Percentage of general government expenditure": "share_of_gov_expenditure",
+    "US dollars per person, PPP converted": "usd_per_person_ppp",
+}
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("social_expenditure")
+
+    # Read table from meadow dataset.
+    tb = ds_meadow.read("social_expenditure")
+
+    #
+    # Process data.
+    #
+    tb = geo.harmonize_countries(
+        df=tb, countries_file=paths.country_mapping_path, excluded_countries_file=paths.excluded_countries_path
+    )
+
+    # Pivot table to have indicator as columns.
+    tb = tb.pivot(
+        index=["country", "year", "expenditure_source", "spending_type", "programme_type_category", "programme_type"],
+        columns="indicator",
+        values="value",
+        join_column_levels_with="_",
```

Comment:
omg i didn't know `join_column_levels_with` was a thing. thanks

Reply:
Yeah, it's cool! Kudos, @pabloarosado 👑

Reply:
One of the hidden gems in our Tables!